arviz/plots/bpvplot.py

```diff
@@ -93,6 +96,8 @@ def plot_bpv(
         For example, `data_pairs = {'y' : 'y_hat'}`
         If None, it will assume that the observed data and the posterior/prior
         predictive data have the same variable name.
+    labeller : labeller instance, optional
+        Class providing the method `make_label_vert` to generate the labels in the plot titles.
```

Comment:
Do we want to allow only labeller objects? Or strings too and have some kind of mapping/`getattr(az.labels, string)`?